apps/codeforafrica/src/components/ProjectDetails/ProjectDescription.js

```diff
@@ -19,15 +21,17 @@ const ProjectDescription = React.forwardRef(
         ref={ref}
       >
         <RichTypography variant="h5">{title}</RichTypography>
-        <RichTypography variant="body1" sx={{ typography: { md: "body2" } }}>
-          {children}
-        </RichTypography>
+        <RichText
+          variant="body1"
+          elements={description}
+          typographyProps={{ sx: { typography: { md: "body2" } } }}
+        />
         {links?.length > 0 ? (
           <Stack direction="row" spacing={2.5}>
-            {links.map(({ content, href, slug }) => (
-              <Button key={slug} href={href} slug={slug}>
-                {content}
-              </Button>
+            {links.map(({ label, href, type }) => (
+              <ProjectDescriptionButton key={href} href={href} slug={type}>
```

Comment:
```suggestion
              <ProjectDescriptionButton key={href} href={href} type={type}>
```

Lets stick to `type`.